src/main/java/com/github/checkstyle/generatepatchfile/GeneratePatchFile.java

```diff
@@ -172,7 +167,7 @@ public void generatePatch(int runPatchNum) throws Exception {
      * @throws Exception exception
      */
     public void generatePatchWithGitCommand(int runPatchNum, String patchFormat) throws Exception {
-        for (int i = 1; i < runPatchNum; i++) {
+        for (int i = 1; i < runPatchNum + 1; i++) {
```

Comment:
why did you increase amount of patches? please move to a new commit if it's not related to new `diff` patch format

Reply:
done